.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1,9 +1,9 @@
 ---
 blank_issues_enabled: false
```

Comment:
You might copy the ignore settings we use in the main repo. In particular CI Yaml files are very susceptible to breaking when reformatted.

Reply:
Happy to. Do we know of any actual issues prettier has created for us in yml? I've used prettier in a few repos now and have not had issues formatting yml (in fact, it has caught issues for me).